platform/security/pdp/security-pdp-authzrealm/src/test/java/ddf/security/pdp/realm/xacml/XacmlPdpTest.java

```diff
@@ -203,77 +232,77 @@ public void testResourceIsNotPermitted() {
 
     @Test
     public void testParseAttributeTypeBoolean() {
-        assertTrue(XACMLConstants.BOOLEAN_DATA_TYPE.equals(testRealm.getXacmlDataType("true")));
+        assertTrue(BOOLEAN_DATA_TYPE.equals(testRealm.getXacmlDataType("true")));
```

Comment:
❓ Should we replace those `assertTrue` with `assertThat(..., is(...))` while we're at it?